lighttpd.conf

```diff
@@ -0,0 +1,58 @@
+server.modules += ( "mod_access",
+                    "mod_auth",
+                    "mod_fastcgi",
+                    "mod_alias",
+                    "mod_rewrite",
+                    "mod_openssl",
+                    "mod_accesslog",
+                    "mod_dirlisting")
+
+server.feature-flags = ( "server.network-mptcp" => "enable" )
+server.use-ipv6 = "enable"
+
+$SERVER["socket"] == "[::]:443" {
+    ssl.engine = "enable"
+    ssl.pemfile = "/etc/letsencrypt/live/check.mptcp.dev/fullchain.pem"
+    ssl.privkey = "/etc/letsencrypt/live/check.mptcp.dev/privkey.pem"
+}
+
+$SERVER["socket"] == "0.0.0.0:443" {
+    ssl.engine = "enable"
+    ssl.pemfile = "/etc/letsencrypt/live/check.mptcp.dev/fullchain.pem"
+    ssl.privkey = "/etc/letsencrypt/live/check.mptcp.dev/privkey.pem"
+}
+
+$SERVER["socket"] == "[::]:80" {}
+$SERVER["socket"] == "0.0.0.0:80" {}
+
+$HTTP["url"] =~ "^/\.well-known/acme-challenge/" {
+    server.document-root = "/var/www"
+}
+
+$HTTP["url"] =~ "^/files/" {
+    dir-listing.activate = "enable"
+    server.document-root = "/var/www/"
+}
+
+server.document-root = "/var/www/"
+server.upload-dirs = ( "/var/cache/lighttpd/uploads" )
+server.errorlog = "/var/log/lighttpd/error.log"
+
+debug.log-request-handling  = "enable"
+accesslog.filename = "/var/log/lighttpd/access.log"
+
+fastcgi.server = (
+    "/app.fcgi" =>
+    ((
+        "socket" => "/tmp/app-fcgi.sock",
+        "bin-path" => "/flask_app/app.fcgi",
+        "check-local" => "disable",
+        "max-procs" => 1,
+    ))
+)
+
+url.rewrite-once = (
+ "^(/files.*)$" => "$1",
```

Comment:
linked to my comment above: maybe that causing the extra trailing `/`?

Reply:
I do not see why it would